docs/add-non-um-users.html

```diff
@@ -3,5 +3,37 @@
 title: Add Non-UM Users
 ---
 
-<h1 class="mt-5">Add Non-UM Users Documentation</h1>
-<p>Follow documentation for this Feature</p>
\ No newline at end of file
+<h1 class="mt-5">Add Non-UM Users</h1>
```

Comment:
```suggestion
<h1 class="mt-5">Add Non U-M Users</h1>
```

Reply:
changing heading so it is U-M to follow established conventions

Reply:
@melindakraft This is how it is show in CCM app, I will stick with what I have. 
![Screen Shot 2022-04-15 at 4 02 14 PM](https://user-images.githubusercontent.com/8579775/163626731-5e5b1ae7-07a7-4fbe-a1b6-ef29651de0d8.png)